src/main/frontend/src/components/landingpage/InfiniteScrollComponent.js

```diff
@@ -0,0 +1,82 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import InfiniteScroll from "react-infinite-scroll-component";
+import { useEffect, useState } from "react";
+import LandingPageItem from "./LandingPageItem";
+import { Row } from "react-bootstrap";
+
+function InfiniteScrollComponent() {
+  const [items, setItems] = useState([]);
+
+  const [noMore, setnoMore] = useState(true);
+
+  const [page, setpage] = useState(2);
+
+  const [divHelper, setdivHelper] = useState(0);
+
+  useEffect(() => {
+    const getItems = async () => {
+      const res = await fetch(
+        `http://localhost:8080/api/v1/items?page=1&size=20`
+      );
+      const data = await res.json();
+
+      setItems(data["content"]);
+    };
+
+    getItems();
```

Comment:
you can avoid this by defining useEffects callback function as async: 
```suggestion
   useEffect(async () => {})
```